core/src/test/scala/integration/kafka/api/BaseConsumerTest.scala

```diff
@@ -82,112 +86,19 @@ abstract class BaseConsumerTest extends IntegrationTestHarness with Logging {
   }
 
   @Test
-  def testAutoCommitOnRebalance() {
```

Comment:
Did we move all of these to the plaintext test just to save test time since there's actually not much that needs to be exercised to validate the security functionality?